assets/src/scripts/services/state/water-levels.js

```diff
@@ -45,6 +48,20 @@ export const retrieveWaterLevels = (agencyCode, siteID) => (dispatch) => {
  */
 export const getWaterLevels = state => state[MOUNT_POINT];
 
+/**
+ * Return water level data for the site or the empty object if no data available
+ * @param {String} agencyCode
+ * @param {String} siteID
+ * @return {Function} selector to return water level data for the site. Returns empty object if no data available.
+ */
+export const getSiteWaterLevels = memoize((agencyCode, siteID) =>  createSelector(
```

Comment:
This is something I'm thinking about for the well log work I'm starting now... I think we need a good way for multiple components to independently declare what service data they need, without it triggering a second service call.

Reply:
Yeah, I realize this is somewhat cheating in that I'm assuming the data is already there.